packages/core/src/batch/BatchRenderer.ts

```diff
@@ -295,7 +295,7 @@ export class BatchRenderer extends ObjectRenderer
     get MAX_TEXTURES(): number
     {
         // #if _DEBUG
-        deprecation('7.1.0', 'PIXI.BatchRenderer.MAX_TEXTURES renamed to PIXI.BatchRenderer.maxTextures');
+        deprecation('7.1.0', 'BatchRenderer#MAX_TEXTURES renamed to BatchRenderer#maxTextures');
```

Comment:
This should be `renamed to BatchRenderer.defaultMaxTextures`?
Also there are '#' instead of '.'

Reply:
ahh! 

got used to seeing static properties in these PR's 😄

Reply:
Threw in this change for free 😝

Reply:
TITLE_CASING for members is not great